src-files/def-config.yaml

```diff
@@ -25,15 +25,24 @@ verification:
 
   # File prefixes, references in which should not be analyzed.
   notScanned:
-    # Github-specific files
+    # GitHub-specific files
     - .github/pull_request_template.md
     - docs/pull_request_template.md
 
-  # Glob patterns describing the files which do not physically exist in the repository
-  # but should be treated as existing nevertheless.
+    # GitLab-specific files
+    - .gitlab/merge_request_templates/
```

Comment:
Also there can be `.gitlab/issues_templates`. But I would check whether local references work fine there first. Also I suppose we don't want to check only local references in MR templates, but we want to check global references, right? How does it work currently?

Reply:
Currently, it's only possible to ignore a file entirely. Which seems good in our case because templates often contain incomplete links like `https://issues.serokell.io/issue/INT-`.

> But I would check whether local references work fine there first
Do you know a repository where I could spoil `master` to check that? :peka:

Reply:
>  UPD: actually I can try in this repository and then just force-push to master.

Meh, it's better not to do it, because in some repos we use `master` of this repo in CI (not a specific commit). What about this one https://github.com/Martoon-00/Natural-science-X? (I've been using my old personal repo for testing previously.) You can force-push there, probably chances that it will be noticed are smaller than for `crossref-verifier`.

Reply:
Or maybe https://github.com/Martoon-00/HW-10, it has less contributors.

Reply:
Hm, apparently you don't even have to force-push.

Reply:
That's true :joy: :joy: :joy: 

Okay, I've found out that if I want to refer to some repository file from issue, it should be of form `../blob/master/<absolute path to file>`. So apparently crossref-verifier cannot handle such for now.

Reply:
> Hm, apparently you don't even have to force-push.

Reply:
Oh, wait.
> Also there can be .gitlab/issues_templates. But I would check whether local references work fine there first. 

Here I was taking about **GitLab**.

Reply:
Oh mda, right.

Well, somehow it happens that GitLab also required `../blob/master/README.md` link if I put it to issue, so I'm adding.